examples/applicationCommands.js

```diff
@@ -0,0 +1,103 @@
+const Eris = require("eris");
+
+const Constants = Eris.Constants;
+
+// Replace TOKEN with your bot account's token
+const bot = new Eris("BOT_TOKEN", {
+    intents: [
+        "guilds"
+    ]
+});
+
+bot.on("ready", async () => { // When the bot is ready
+    console.log("Ready!"); // Log "Ready!"
+
+    //Note: You should use guild commands to test, as they update instantly. Global commands can take up to an hour to update.
+
+    const commands = await bot.getCommands();
+
+    if(!commands.length) {
+        bot.createCommand({
+            name: "test_chat_input",
+            description: "Test command to show how to make commands",
+            options: [ //An array of Chat Input options https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-structure
+                {
+                    "name": "animal", //The name of the option
+                    "description": "The type of animal",
+                    "type": Constants.CommandOptionTypes.STRING, //This is the type of string, see the types here https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-type
+                    "required": true,
+                    "choices": [ //The possible choices for the options
+                        {
+                            "name": "Dog",
+                            "value": "animal_dog"
+                        },
+                        {
+                            "name": "Cat",
+                            "value": "animal_cat"
+                        },
+                        {
+                            "name": "Penguin",
+                            "value": "animal_penguin"
+                        }
+                    ]
+                },
+                {
+                    "name": "only_smol",
+                    "description": "Whether to show only baby animals",
+                    "type": Constants.CommandOptionTypes.BOOLEAN,
+                    "required": false
+                }
+            ],
+            type: Constants.CommandTypes.CHAT_INPUT //Not required for Chat input type, but recommended
+        }); //Create a chat input command
+
+        bot.createCommand({
+            name: "Test User Menu",
+            type: Constants.CommandTypes.USER
+        }); //Create a user context menu
+
+        bot.createCommand({
+            name: "Test Message Menu",
+            type: Constants.CommandTypes.MESSAGE
+        }); //Create a user context menu
```

Comment:
I guess this was meant to be message right?
```suggestion
        }); //Create a message context menu
```

Reply:
Oop yea I'm stupid, when I changed the wording I forgot to change that